src/commands/service/get-display-name.md

```diff
@@ -0,0 +1,4 @@
+## Description
+
+Gets display name of a service based on the provided path by extracting this
+information from bedrrock.yaml
```

Comment:
Gets display name of a service based on the provided path by extracting this
information from `bedrock.yaml`. This command tries to locate `bedrock.yaml` in the current folder, ...

please add more information like what happen of `bedrock.yaml` is not found, or have it as pre requisite that `bedrock.yaml` has to be in the current directory.

Reply:
@dennisseah Done. Please note that this command is not going to be useful for anyone to run manually (it's trivial), it's for the pipeline to run.